src/styles/variables.scss

```
@@ -0,0 +1,6 @@
+$gap: 1px;
+$day-size: 100px;
+$padding: 10px;
+$days: "mon" 0, "tue" $day-size + $gap, "wed" ($day-size * 2) + ($gap * 2),
+  "thu" ($day-size * 3) + ($gap * 3), "fri" ($day-size * 4) + ($gap * 4),
+  "sat" ($day-size * 5) + ($gap * 5), "sun" ($day-size * 6) + ($gap * 6);
```

improve sizes with loop
```suggestion
$days: 
  "mon" 0, 
  "tue" $day-size + $gap, 
  "wed" ($day-size * 2) + ($gap * 2),
  "thu" ($day-size * 3) + ($gap * 3), 
  "fri" ($day-size * 4) + ($gap * 4),
  "sat" ($day-size * 5) + ($gap * 5), 
  "sun" ($day-size * 6) + ($gap * 6);
```